Makefile.helper.mk

```diff
@@ -2,13 +2,11 @@
 # SRO-specific options
 
 SPECIALRESOURCE  ?= driver-container-base
-NAMESPACE        ?= openshift-special-resource-operator
+NAMESPACE        ?= special-resource-operator
 PULLPOLICY       ?= IfNotPresent
 TAG              ?= $(shell git rev-parse --abbrev-ref HEAD)
 CSPLIT           ?= csplit - --prefix="" --suppress-matched --suffix-format="%04d.yaml"  /---/ '{*}' --silent
 YAMLFILES        ?= $(shell  find manifests charts -name "*.yaml"  -not \( -path "charts/lustre/lustre-aws-fsx-0.0.1/csi-driver/*" -prune \)  -not \( -path "charts/*/shipwright-*/*" -prune \) -not \( -path "charts/experimental/*" -prune \) )
-PLATFORM         ?= ""
```

Comment:
I don't think we should remove the PLATFORM here.

I understand it has nothing to do with upstream, but I still think we should keep it as an option, the reasons are:
1. There is nothing wrong with supporting OCP upstream IMO
2. I think it will be better to minimize the u/s d/s diff and in this case we can have a smaller diff.

With that said, I do think we should change the default behaviour to k8s instead of OCP but keep `PLATFROM=ocp` as an option

Reply:
But `PLATFORM` serves no purpose - it was merely here to support a suffix to the `manifests` directory. Upstream after this PR remains compatible with OCP.

Reply:
I confused with `SUFFIX`, so as long as we keep it and just change the default to k8s I am OK with that.

Reply:
I claim there's no need for `SUFFIX` either 🙂